gulpfile.js

```diff
@@ -0,0 +1,18 @@
+var gulp = require('gulp');
```

Comment:
I think gulp is easier to understand and maintain. Your choice if you want to reimplement my work.